### PR TITLE
Some CONFIGURE left, mainly in test comment.

### PR DESCRIPTION
**Pull request by aHenryJard**
See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
* ...

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### CODE REVIEW

File: core/src/main/java/jenkins/model/Jenkins.java [hunk 1]
```diff
@@ -1911,7 +1911,7 @@ public View getPrimaryView() {
      }
 
     public void setPrimaryView(@Nonnull View v) {
-        checkPermission(Permission.CONFIGURE);
+        checkPermission(Jenkins.ADMINISTER);
```
Comment (EstherAF) [inline core/src/main/java/jenkins/model/Jenkins.java:1914]: Is this in the JEP? that the primary view is only allowed to administer?

File: test/src/test/java/hudson/cli/ReloadConfigurationCommandTest.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ public void reloadConfigurationShouldFailWithoutAdministerOrConfigurePermission(
     public void reloadConfigurationShouldWorkWithConfigurePermission() throws Exception {
         j.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy()
                                                    .grant(Jenkins.MANAGE, Jenkins.READ).everywhere().toAuthenticated());
-        //Any reload configuration should work with CONFIGURE_JENKINS as well
+        //Any reload configuration should work with _JENKINS as well
```
Comment (EstherAF) [inline test/src/test/java/hudson/cli/ReloadConfigurationCommandTest.java]: `_JENKINS`?

**Review by EstherAF — CHANGES_REQUESTED** [review]

File: core/src/main/java/jenkins/model/Jenkins.java (hunk 1 above)
Comment (aHenryJard) [reply core/src/main/java/jenkins/model/Jenkins.java:1914]: no it's the last ongoing discussion on the PR https://github.com/jenkinsci/jenkins/pull/4374#discussion_r350830926
Anyway I think that Permission.CONFIGURE is a mistake, it should be Jenkins.ADMINISTER or Jenkins.MANAGE, but reading the discussion on the PR I think it's for ADMINISTER.

File: test/src/test/java/hudson/cli/ReloadConfigurationCommandTest.java (hunk 2 above)
Comment (aHenryJard) [reply test/src/test/java/hudson/cli/ReloadConfigurationCommandTest.java]: oops XD thank you

Comment (aHenryJard) [reply test/src/test/java/hudson/cli/ReloadConfigurationCommandTest.java]: done.

**Review by EstherAF — APPROVED** [review]

**Comment by EstherAF** [comment]
However, in the JEP, the setPrimaryView appears as CONFIGURE/MANAGE, so we should change that there as well.

**Pull request merged**